jenkinsapi/fingerprint.py

```diff
@@ -81,10 +82,10 @@ def validate_for_build(self, filename: str, job: str, build: int) -> bool:
             if self._data["original"]["name"] == job:
                 if self._data["original"]["number"] == build:
                     return True
-        if self._data["fileName"] != filename:
+        if self._data["hash"] != self.id_:
             log.info(
```

Comment:
This `log.info()` call should likely be updated to the following:
```
log.info(
    "File hash from Jenkins (%s) did not match local hash (%s)",
    self._data["hash"], self.id_
)
```
I did not update the `log.info()` call in the spirit of making a minimal change.